src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -26,24 +26,25 @@ an output file's bitrate in the same operation as a file conversion. For this
 cheat sheet, I will show these operations (among others) as separate commands
 for the sake of clarity.
  
-Please let me know of useful additions or corrections. [Pull requests are welcome](https://github.com/google/WebFundamentals).
+Please let me know of useful additions or corrections.
+[Pull requests are welcome](https://github.com/google/WebFundamentals/tree/master/src/content/en/fundamentals/media/manipulating/cheatsheet).
 
 ## Display characteristics
 
     packager input=myvideo.mp4 --dump_stream_info
 
     ffmpeg -i myvideo.mp4
 
-This is technically an incorrect usage of ffmpeg. As such it prints an error
-message to the screen. It does however list information not available from the
-Shaka Packager command.
+Technically, ffmpeg always requires an output file format. Calling ffmpeg this
+way will give you an error message explaining that; however, it lists
+information not available from the Shaka Packager command.
 
 ## Demux (split audio and video)
 
-My preference would be to show file type conversion before diving into demuxing.
-It appears that Shaka Packager cannot do conversion without also demuxing.
+Most of you wouldn't care about this, but Shaka Packager requires demuxing in
```

Comment:
"...requires demuxing (splitting the audio and video into separate files) in order..."

I realized my problem with the debate above is that "Demux (split audio and video files)" is ambiguous - demux is a term you need to define, and "split audio and video files" can mean "split[ting] audio and video files" (i.e., verb and noun), or it can mean "audio and video files that have been split" (i.e., adjective and noun).  I think expanding the reference to demux fixes this without getting in to your referenced debate.  (EVEN THOUGH YOU'RE ALL WRONG.  :) )